includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -214,6 +215,19 @@ public function show_error_if() {
 			</div>
 			<?php
 		}
+
+		// Display classify post success message for manually classified post.
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$classified = isset( $_GET['classified'] ) ? intval( wp_unslash( $_GET['classified'] ) ) : 0;
```

Comment:
`classified` should be added to the `removable_query_args` filter.

```
add_filter( 'removable_query_args', function( $removable_query_args ) {
   $removable_query_args[] = 'classified';
  return $removable_query_args;
} );
```

to avoid the potential for a collision, it might be worth being more specific with something like `classifai_reclassify` -- it's best to namespace all the things, not just function names :)